test/arrays.jl

```diff
@@ -72,6 +72,15 @@ getdef(v) = getmetadata(v, Symbolics.VariableDefaultValue)
 
     # #417
     @test isequal(Symbolics.scalarize(x', (1,1)), x[1])
+
+    test_mat = [1 2; 3 4]
+    repl_dict = Dict(Symbolics.scalarize(B .=> test_mat))
+    @test isequal(substitute(Symbolics.scalarize(B^2), repl_dict), test_mat^2)
+    @test isequal(substitute(Symbolics.scalarize(B^3), repl_dict), test_mat^3)
+    @test isequal(substitute(Symbolics.scalarize(B^4), repl_dict), test_mat^4)
+    @test_broken isequal(substitute(Symbolics.scalarize(B^5), repl_dict), test_mat^5)
+    @test_broken isequal(substitute(Symbolics.scalarize(B^6), repl_dict), test_mat^6)
+    @test_broken isequal(substitute(Symbolics.scalarize(B^7), repl_dict), test_mat^7)
```

Comment:
What is the meaning of "Netsted ArrayOps?" Do these tests directly involve them?